perllib/FixMyStreet/App/Controller/Admin/Users.pm

```diff
@@ -321,7 +322,10 @@ sub edit : Chained('user') : PathPart('') : Args(0) {
         $user->phone_verified( $phone_v );
         $user->name( $name );
 
-        $user->flagged( $c->get_param('flagged') || 0 );
+        # Only superusers can flag / unflag a user
```

Comment:
This might need to go under the superuser or Zurich bit of code bit further down, see other comment; and I guess the new creation too.